networking/bench/Sender/log-config.yaml

```diff
@@ -0,0 +1,44 @@
+# global filter; messages must have at least this severity to pass:
+minSeverity: Debug
+
+# global file rotation settings:
+rotation:
+  rpLogLimitBytes: 5000000
+  rpKeepFilesNum:  10
+  rpMaxAgeHours:   24
+
+# these backends are initialized:
+setupBackends:
+  - KatipBK
+
+# if not indicated otherwise, then messages are passed to these backends:
+defaultBackends:
+  - KatipBK
+  - AggregationBK
```

Comment:
`- AggregationBK` is not needed. It will cause the generation of statistics for logging severity. This maybe a reason why the queue is overfilled.